aio/tools/transforms/templates/cli/lib/cli.html

```diff
@@ -78,20 +81,16 @@
 {% if name.length > 1 %}-{% endif %}-{$ name $}
 {%- endmacro %}
 
-{%- macro renderValues(values, default) -%}
+{%- macro renderValues(values) -%}
 {%- set valString = values.join('|') -%}
-{%- if valString.length > 15 %}<br>    {% endif %}{$ valString $}
+{%- if valString.length > 15 %}{% endif %}{$ valString $}
```

Comment:
Now that the length of `valString` makes no difference this whole macro can be significantly simplified:

```suggestion
{$ values.join('|') $}
```